### ISSUE TITLE
Error importing: Cannot find module 'elegua' or its corresponding type declarations

### ISSUE THREAD

**Issue by conradolandia** [howesteve/elegua#2]
If I install the package either from npm or git, I can't import it. Always gives the error `Error importing: Cannot find module 'elegua' or its corresponding type declarations`.

**Comment by howesteve** [comment]
Did you install it in your project, i.e. `pnpm install elegua`? What version of Elegua are you using?

**Comment by conradolandia** [comment]
I installed it using `npm i -D elegua`. Also tried with pnpm, neither worked. I also tried `npm i -D https://github.com/howesteve/elegua` I didn't specify a version, so the last available one I guess.

**Comment by howesteve** [comment]
It's working here. Do you mean the demo app? I forced elegua as a self dependency. Try with latest version and see if it's better.

**Comment by conradolandia** [comment]
No, I mean installing it on a custom Svelte project. I will try again and if I get the error I will paste it here.

**Comment by slavnycoder** [comment]
Same
![image](https://github.com/howesteve/elegua/assets/13649915/25451676-01e0-478a-8250-f6dea4e69215)
![image](https://github.com/howesteve/elegua/assets/13649915/4a215761-5a47-44ed-932e-01db56d3e078)

**Comment by howesteve** [comment]
It's working here. Can anyone provide me a repository where I could replicate the situation? Or just try the steps below?

```
gargamel::~/ttt $ git clone https://github.com/howesteve/elegua
Cloning into 'elegua'...
remote: Enumerating objects: 219, done.
remote: Counting objects: 100% (219/219), done.
remote: Compressing objects: 100% (128/128), done.
remote: Total 219 (delta 103), reused 181 (delta 65), pack-reused 0
Receiving objects: 100% (219/219), 80.28 KiB | 6.17 MiB/s, done.
Resolving deltas: 100% (103/103), done.
gargamel::~/ttt $ cd elegua
gargamel::~/ttt/elegua ‹master› $ pnpm install
Lockfile is up to date, resolution step is skipped
Packages: +201
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
Packages are hard linked from the content-addressable store to the virtual store.
  Content-addressable store is at: /home/howe/.local/share/pnpm/store/v3
  Virtual store is at:             node_modules/.pnpm
Progress: resolved 201, reused 201, downloaded 0, added 201, done

dependencies:
+ svelte 3.59.1

devDependencies:
+ @sveltejs/adapter-auto 2.1.0
+ @sveltejs/kit 1.20.4
+ @sveltejs/package 2.0.2
+ @sveltejs/vite-plugin-svelte 2.4.1
+ @typescript-eslint/eslint-plugin 5.59.11
+ @typescript-eslint/parser 5.59.11
+ elegua 1.0.7
+ eslint 8.43.0
+ eslint-config-prettier 8.8.0
+ eslint-plugin-svelte3 4.0.0
+ prettier 2.8.8
+ prettier-plugin-svelte 2.10.1
+ svelte-check 3.4.3
+ tslib 2.5.3
+ typescript 5.1.3
+ vite 4.3.9

Done in 1.4s
gargamel::~/ttt/elegua ‹master› $ pnpm run dev

> elegua@1.0.8 dev /home/howe/ttt/elegua
> echo Starting demo application... && vite dev

Starting demo application...
Forced re-optimization of dependencies
Port 5173 is in use, trying another one...

  VITE v4.3.9  ready in 719 ms

  ➜  Local:   http://localhost:5174/
  ➜  Network: use --host to expose
  ➜  press h to show help
```

**Comment by conradolandia** [comment]
Your demo app works. Sorry for the trouble, man. I got my own working. I had to copy the file `Elegua.svelte` from your demo app to my project's `lib/` folder, and it finally worked using `import Router, { path, url, hash } from './lib/Elegua.svelte';`, in a similar way as you load it on your demo app. However, your README file says to use `import Route, { params, matches } from 'elegua';`, I guess to load it as a module from `node_modules`? which doesn't work. VSCode complains that the module can´t be found. There are no errors on the console, but the Router is not rendered importing it this way. I've tried installing Elegua with `npm` and `pnpm`, multiple times, closely following the README.
